homework-20-12-23/sos-ghazaryan/cleanup.sh

```diff
@@ -27,7 +27,7 @@ function cleanInstancesByTag
 #   Loop through each instance and terminate it
   for instance_id in $instances_to_delete; do
       echo "Terminating instance $instance_id"
-      aws ec2 terminate-instances --instance-ids $instance_id
+      aws ec2 wait terminate-instances --instance-ids $instance_id
```

Comment:
`aws ec2 wait terminate-instances` does not perform instance deletion, you need to terminate the instance first, and then use this to wait for termination completion

Reply:
aws ec2 terminate-instances --instance-ids $instance_id with wait was fails reason i change the code like this
`aws ec2 terminate-instances --instance-ids $instance_id
      aws ec2 wait instance-terminated --instance-ids $instance_id`